### ISSUE TITLE
6×6 stiffness matrix

### ISSUE THREAD

**Issue by lzg38844037** [sandialabs/NuMAD#57]
Dear Sir, hello
Can NuMAD calculate the 6×6 stiffness matrix of a wind turbine blade section?

**Comment by ecamare** [comment]
Yes but the input is not a 2D cross-sectional mesh. The input would be the complete shell mesh of the beam/wind blade. It would use Blade Properties Extractor (BPE).

**Comment by lzg38844037** [comment]
Hello, Sir. 
Is the calculation result of BPE accurate? Does version 2 of NuMAD support BPE computing? 
I look forward to your reply. Thank you
